handlers/main.yml

```diff
@@ -19,6 +19,6 @@
   become: True
```

Comment:
May be better use REST API calls by `uri` module instead of `commad`?

Reply:
No, this is breaking change